helm/cluster-openstack/files/run/kubeadm/gs-kube-proxy-config.yaml

```diff
@@ -0,0 +1,4 @@
+---
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
+kind: KubeProxyConfiguration
+metricsBindAddress: "0.0.0.0:10249"
```

Comment:
What do you think about adding the rest of our standard configuration from `k8scloudconfig`? https://github.com/giantswarm/k8scloudconfig/blob/master/files/config/kube-proxy.yaml Maybe everything but `conntrack`. `kubectl-gs` already has a [pods-cidr flag](https://github.com/giantswarm/kubectl-gs/blob/master/cmd/template/cluster/flag.go#L52) which would be cool to wire up.

Reply:
Created an issue for that https://github.com/giantswarm/roadmap/issues/790